torch_frame/testing/image_embedder.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import torch
+from PIL import Image
+from torch import Tensor
+
+
+class RandomImageEmbedder:
```

Comment:
We can inherit the parent class so that all you need to write is `forward_embed`